Dockerfile

```diff
@@ -1,15 +1,19 @@
-FROM node:16.20.0
+# Bullseyes uses Debian 11 with longer end-of-life date. Slim variant means smaller software component
+FROM node:16.20.0-bullseye-slim as build
 
-WORKDIR /usr/src/app
+RUN apt-get update && apt-get install -y --no-install-recommends dumb-init
+
+ENV NODE_ENV production
 
-COPY package*.json ./
+WORKDIR /usr/src/app
 
-RUN npm install
+COPY --chown=node:node . /usr/src/app
 
-COPY . .
+RUN npm ci --only=production
 
```

Comment:
I am wondering whether we could further improve it by using a multi-stage build (s. [here](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#use-multi-stage-builds) and [here](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#use-multi-stage-builds)).

Reply:
Should be in this commit [`0961e8e` (#21)](https://github.com/digitalservicebund/useid-eservice-example/pull/21/commits/0961e8e4ffb250271dbbe0d0f1b31f07f4fc54cf)